bokehjs/src/lib/models/widgets/datetime_picker.ts

```diff
@@ -0,0 +1,63 @@
+import flatpickr from "flatpickr"
+
+import {BaseDatetimePicker, BaseDatetimePickerView} from "./base_datetime_picker"
+import {DateLike} from "./base_date_picker"
+import {assert} from "core/util/assert"
+import * as p from "core/properties"
+
+export class DatetimePickerView extends BaseDatetimePickerView {
+  declare model: DatetimePicker
+
+  protected override get flatpickr_options(): flatpickr.Options.Options {
+    return {
+      ...super.flatpickr_options,
+      mode: "single",
```

Comment:
I think datetime widgets should enable `minDateHasTime` and `maxDateHasTime`. This will make the start/end datetimes and not only dates.

https://github.com/holoviz/panel/blob/aae2074941b73fd3d8e2dbad4e2c68acfd70b968/panel/models/datetime_picker.ts#L101-L102

Reply:
I'm addressing this [in](https://github.com/bokeh/bokeh/pull/12792#discussion_r1108341110).